packages/okta-adapter/src/filters/policy_priority.ts

```diff
@@ -114,9 +143,9 @@ const createPolicyRulePriorityInstance = ({
   })
 }
 
-// For AccessPolicy, the priority index starts from 1, while for others it starts from 0.
-const setPriority = (typeName: string, priority: number): number => {
-  if (typeName === ACCESS_POLICY_TYPE_NAME) {
+// For AccessPolicyRules, the priority index starts from 0, while for others it starts from 1.
+export const setPriority = (typeName: string, priority: number): number => {
```

Comment:
probably leftover from the older version :) same for `deployPriorityChange`

```suggestion
const setPriority = (typeName: string, priority: number): number => {
```